apis/v1alpha1/types.go

```diff
@@ -973,11 +1034,26 @@ type ModelPackageContainerDefinition struct {
 	ModelDataURL      *string `json:"modelDataURL,omitempty"`
 }
 
+// A group of versioned models in the model registry.
+type ModelPackageGroup struct {
+	CreationTime *metav1.Time `json:"creationTime,omitempty"`
```

Comment:
I see the `CreationTime` field has been added to a whole bunch of resources and type definitions... is this because you are using a new(er) version of aws-sdk-go to build the sagemaker-controller? If so, it would be best to regenerate the sagemaker-controller with that new aws-sdk-go version but *without* adding the new FeatureGroup resource. That way, we can isolate the changes in this PR to only those introduced due to the new FeatureGroup resource. Make sense?

Reply:
The funny thing is... I don't actually see a change to the `go.mod` file in this PR... @BriannaRoskind did you happen to regenerate the sagemaker-controller and pass in a specific aws-sdk-go version to generate the controller with but forgot to change the `go.mod` file in the sagemaker-controller repo?